homeassistant/components/anglian_water/strings.json

```
@@ -0,0 +1,25 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Fill in your Anglian Water login information",
+        "data": {
+          "username": "[%key:common::config_flow::data::email%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        }
+      }
+    },
+    "error": {
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "maintenance": "Anglian Water is currently performing maintenance, please try again later"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "previous_consumption": {
+        "name": "Previous Consumption"
```

```suggestion
        "name": "Previous consumption"
```
What does previous mean in this context? Previous day? Previous week? Previous cycle?

---

previous day, I will update to "previous day" instead

---

No sorry, `Yesterday consumption` that sounds better.

---

Consumption yesterday?